ci/gitlab/.gitlab-ci.yml

```diff
@@ -5,6 +5,7 @@ stages:
   - oe-checkout
   - oe-test
   - deploy
+  - build downstream
```

Comment:
Maybe the stage can be renamed "trigger" and we can add others there when they arise (sounds like qa wants that too)